### PR TITLE
Update README.md

### LINKED ISSUE #424: How to contribute guidelines

### PR DESCRIPTION
**Pull request by thealmarty**
- per https://github.com/metastatedev/community/issues/71
- please also read the raw code because there are comments that are invisible in the rendered version.
- there are outstanding comments/holes to fill in (blocked on other materials).
- closes #424

### CODE REVIEW

File: README.md
Comment (cwgoes) [inline README.md:94]:
```suggestion
Install the [Juvix package](https://marketplace.visualstudio.com/items?itemName=metastate.language-juvix)
```

File: README.md
Comment (cwgoes) [inline README.md:155]:
```suggestion
## License: GPL 3.0
```

We can link to `LICENSE`

File: README.md
Comment (cwgoes) [inline README.md:153]: Let's use a separate file for this

File: README.md
Comment (cwgoes) [inline README.md:101]: This doesn't really work yet, let's leave it out for now

**Review by cwgoes — COMMENTED** [review]
Thanks! A few comments.

File: README.md
Comment (thealmarty) [inline README.md:141]: Perhaps @awasunyin or @alexfmpe can help fill out this section?

File: README.md
Comment (thealmarty) [reply README.md:144]: Sure. Where would you like the file?

**Comment by Asamartino** [comment]
In _Installation_ > _Instructions for Linux_: will it be possible to add instructions for Windows? 
As there are instructions for macOS will it be possible to rename this subsection "_Instructions_" ?

File: README.md
Comment (cwgoes) [reply README.md:144]: CREDITS or CREDITS.md is fine

**Comment by cwgoes** [comment]
> In _Installation_ > _Instructions for Linux_: will it be possible to add instructions for Windows?
> As there are instructions for macOS will it be possible to rename this subsection "_Instructions_" ?

At the moment no Juvix team members have Windows. Theoretically `stack` should work just fine.

**Comment by thealmarty** [comment]
> In _Installation_ > _Instructions for Linux_: will it be possible to add instructions for Windows?
> As there are instructions for macOS will it be possible to rename this subsection "_Instructions_" ?

I've updated the installation section. Could someone with Windows confirm that the `stack` official instructions work? Thanks.

File: README.md
Comment (awasunyin) [reply README.md:139]: For now the subscribe feature is implemented on our website, let's just add a reference to the official website juvix.org and they'll be able to find the subscribe button there.

**Review by awasunyin — COMMENTED** [review]
lgtm

**Comment by cyberglot** [comment]
so GitHub now has "dark-mode", could we have a version of tara without background? potentially with some nice borders around her so we can see her pointer better? ^___^""

Screenshot 2021-01-12 at 14 24 36

**Comment by Asamartino** [comment]
I had a problem during installation on Ubuntu (using a virtual machine).  See below:
juvix

Jeremy found the solution. I had to increase the RAM to 8GB (as explained [here](https://www.reddit.com/r/haskell/comments/cmx7dl/build_cabal_with_stack_build_out_of_memory_in/)).
I think we should consider mentioning it somewhere.

**Comment by thealmarty** [comment]
Please help fill out CREDITS.md @cwgoes thanks.

File: README.md
Comment (thealmarty) [reply README.md]: Added in doc/, still to fill out.

**Comment by thealmarty** [comment]
I removed the *screenshots* section and the links to tutorials and documentations so that we can merge as is. We can add those later. Let's merge this soon.

**Comment by cwgoes** [comment]
> Please help fill out CREDITS.md @cwgoes thanks.

Updated; it's fine for now, the language reference citation list is pretty extensive.

**Review by cwgoes — APPROVED** [review]
ACK, let's wait for @awasunyin approval to merge

**Review by awasunyin — APPROVED** [review]
Yeah, lgtm. After this is merged, @Asamartino can go ahead and change the mascot for the actual logo from Juvix.

**Review by Asamartino — APPROVED** [review]

**Comment by Asamartino** [comment]
nitpick: license GPL-3.0 is mentioned twice (at the top and at the bottom). Should we remove one of them ?

**Comment by thealmarty** [comment]
> nitpick: license GPL-3.0 is mentioned twice (at the top and at the bottom). Should we remove one of them ?

Feel free! :)

**Pull request merged**
